syntax = "proto3";

package clashleaders;

message PlayerStats {
  int32 home_barbarian_king = 1;
  int32 home_archer_queen = 2;
  int32 home_grand_warden = 3;
  int32 builderbase_battle_machine = 4;
  int32 home_barbarian = 5;
  int32 home_archer = 6;
  int32 home_goblin = 7;
  int32 home_giant = 8;
  int32 home_wall_breaker = 9;
  int32 home_balloon = 10;
  int32 home_wizard = 11;
  int32 home_healer = 12;
  int32 home_dragon = 13;
  int32 home_pekka = 14;
  int32 home_minion = 15;
  int32 home_hog_rider = 16;
  int32 home_valkyrie = 17;
  int32 home_golem = 18;
  int32 home_witch = 19;
  int32 home_lava_hound = 20;
  int32 home_bowler = 21;
  int32 home_baby_dragon = 22;
  int32 home_miner = 23;
  int32 builderbase_raged_barbarian = 24;
  int32 builderbase_sneaky_archer = 25;
  int32 builderbase_beta_minion = 26;
  int32 builderbase_boxer_giant = 27;
  int32 builderbase_bomber = 28;
  int32 builderbase_super_pekka = 29;
  int32 builderbase_cannon_cart = 30;
  int32 builderbase_drop_ship = 31;
  int32 builderbase_baby_dragon = 32;
  int32 builderbase_night_witch = 33;
  int32 home_wall_wrecker = 34;
  int32 home_battle_blimp = 35;
  int32 home_ice_golem = 36;
  int32 home_electro_dragon = 37;
  int32 home_stone_slammer = 38;
  int32 home_lightning_spell = 39;
  int32 home_healing_spell = 40;
  int32 home_rage_spell = 41;
  int32 home_jump_spell = 42;
  int32 home_freeze_spell = 43;
  int32 home_poison_spell = 44;
  int32 home_earthquake_spell = 45;
  int32 home_haste_spell = 46;
  int32 home_clone_spell = 47;
  int32 home_skeleton_spell = 48;
  int32 home_bat_spell = 49;
  int32 bigger_coffers = 50;
  int32 get_those_goblins = 51;
  int32 bigger_better = 52;
  int32 nice_and_tidy = 53;
  int32 release_the_beasts = 54;
  int32 gold_grab = 55;
  int32 elixir_escapade = 56;
  int32 sweet_victory = 57;
  int32 empire_builder = 58;
  int32 wall_buster = 59;
  int32 humiliator = 60;
  int32 union_buster = 61;
  int32 conqueror = 62;
  int32 unbreakable = 63;
  int32 friend_in_need = 64;
  int32 mortar_mauler = 65;
  int32 heroic_heist = 66;
  int32 league_all_star = 67;
  int32 x_bow_exterminator = 68;
  int32 firefighter = 69;
  int32 war_hero = 70;
  int32 treasurer = 71;
  int32 anti_artillery = 72;
  int32 sharing_is_caring = 73;
  int32 keep_your_village_safe = 74;
  int32 master_engineering = 75;
  int32 next_generation_model = 76;
  int32 un_build_it = 77;
  int32 champion_builder = 78;
  int32 high_gear = 79;
  int32 hidden_treasures = 80;
  int32 games_champion = 81;
  int32 dragon_slayer = 82;
  int32 war_league_legend = 83;
  int32 attack_wins = 84;
  int32 best_trophies = 85;
  int32 best_versus_trophies = 86;
  int32 builder_hall_level = 87;
  int32 town_hall_level = 88;
  int32 town_hall_weapon_level = 89;
  int32 trophies = 90;
  int32 versus_battle_win_count = 91;
  int32 versus_battle_wins = 92;
  int32 versus_trophies = 93;
  int32 war_stars = 94;
  int32 defense_wins = 95;
  int32 donations = 96;
  int32 donations_received = 97;
  int32 exp_level = 98;
  int32 get_those_other_goblins = 99;
  int32 builderbase_hog_glider = 100;
  int32 well_seasoned = 101;
  int32 home_royal_champion = 102;
  int32 shattered_and_scattered = 103;
  int32 home_yeti = 104;
  int32 home_siege_barracks = 105;
  int32 home_sneaky_goblin = 106;
  int32 home_super_giant = 107;
  int32 home_super_barbarian = 108;
  int32 home_super_wall_breaker = 109;
  int32 home_super_archer = 110;
  int32 home_super_valkyrie = 111;
  int32 home_inferno_dragon = 112;
  int32 home_super_witch = 113;
  int32 home_headhunter = 114;
  int32 home_super_minion = 115;
}




